foundation/foundation-upgrade/src/test/java/com/boozallen/aissemble/upgrade/migration/v1_10_0/SparkPipelineServletApiMigrationSteps.java

```diff
@@ -0,0 +1,52 @@
+package com.boozallen.aissemble.upgrade.migration.v1_10_0;
+
+/*-
+ * #%L
+ * aiSSEMBLE::Foundation::Upgrade
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+
+import com.boozallen.aissemble.upgrade.migration.AbstractMigrationTest;
+
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+public class SparkPipelineServletApiMigrationSteps extends AbstractMigrationTest {
+    @Given("a POM with {string} packaging type")
+    public void a_pom_with_packaging_type(String packaging) {
+        if (!packaging.equals("jar")) {
```

Comment:
Q: It doesn't really matter, but curious why you didn't follow the same pattern as the Quarkus migration and just use the clause parameter as part of the file name.